### PR TITLE
Fix/importError #1183

### PR DESCRIPTION
**Pull request by Abelgurung**
Replaced "deprecated_arg_names" with "renamed_arg", it has the same functionality

### CODE REVIEW

File: src/cellrank/pl/_circular_projection.py [hunk 1]
```diff
@@ -69,13 +69,15 @@ def _get_optimal_order(data: Lineage, metric: Metric_T) -> Tuple[float, np.ndarr
 
 
 @d.dedent
-@deprecated_arg_names({"labeldistance": "label_distance", "labelrot": "label_rot"})
+@renamed_arg('labeldistance', 'label_distance')
```
Comment (michalk8) [inline src/cellrank/pl/_circular_projection.py:72]: Let's just remove this, it's been deprecated for very long!

**Review by michalk8 — CHANGES_REQUESTED** [review]
Thanks a lot @Abelgurung , let's remove the usage of `renamed_arg` completely!

File: src/cellrank/pl/_circular_projection.py (hunk 1 above)
Comment (Abelgurung) [reply src/cellrank/pl/_circular_projection.py:72]: Got it!

Comment (michalk8) [reply src/cellrank/pl/_circular_projection.py]: Thanks a lot!

**Review by michalk8 — CHANGES_REQUESTED** [review]
@Abelgurung just noticed linter is failing, could you run `tox -e lint-code`? Afterwards, lgtm!
Re failing tests: it's unrelated, JAX installation is missing for some reason @WeilerP for one of the tests.

**Review by michalk8 — APPROVED** [review]
Thanks a lot, LGTM!

**Comment by zhangdae** [comment]
Thanks for your update :-)

**Pull request merged**